### PR TITLE
CASMINST-5332 - adding secondary containers.conf location for cni warnings

### PR DESCRIPTION
**Pull request by rustydb**
### Summary and Scope

- Fixes: CASMINST-5332
- Relates to: #499 

#### Issue Type

- Bugfix Pull Request

This should eliminate the warning messages for the missing cni plugins

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [ ] I tested this on internal system (if yes, please include results or a description of the test)
- [ ] I tested this on a vshasta system (if yes, please include results or a description of the test)
- [ ] I tested this on a vagrant environment (if yes, please include results or a description of the test)

### CODE REVIEW

File: boxes/ncn-node-images/kubernetes/provisioners/common/install.sh [hunk 1]
```diff
@@ -26,6 +26,16 @@ set -e
 
 . /srv/cray/resources/common/vars.sh
 
+echo "configuriing podman cni secondary location"
```
Comment (heemstra) [inline boxes/ncn-node-images/kubernetes/provisioners/common/install.sh:29]: guriing

**Review by heemstra — APPROVED** [review]

File: boxes/ncn-node-images/kubernetes/provisioners/common/install.sh (hunk 1 above)
Comment (rustydb) [reply boxes/ncn-node-images/kubernetes/provisioners/common/install.sh:29]: Didn't catch that in the #499 PR either.

**Review by cdelatte-hpe — APPROVED** [review]

File: boxes/ncn-node-images/kubernetes/provisioners/common/install.sh (hunk 1 above)
Comment (rustydb) [reply boxes/ncn-node-images/kubernetes/provisioners/common/install.sh]: Fixed, and fixed in 1.3 via #502

**Comment by rustydb** [comment]
Fast-forward Succeeded!

**Comment by rustydb** [comment]
Note that the commit has the wrong JIRA.

**Pull request merged**